net-wireless/nitroshare/patches/nitroshare-0.4.0~git.patchset

```diff
@@ -0,0 +1,32 @@
+From e54437dedabadc030692262da6bffb0a153f3f96 Mon Sep 17 00:00:00 2001
+From: Ken Mays <kmays2000@gmail.com>
+Date: Sat, 19 Nov 2022 06:23:12 +0100
+Subject: Fix header locations for HAIKU
+
+
+diff --git a/libnitroshare/src/util/signalnotifier.cpp b/libnitroshare/src/util/signalnotifier.cpp
+index 96fdff3..734a3ac 100644
+--- a/libnitroshare/src/util/signalnotifier.cpp
++++ b/libnitroshare/src/util/signalnotifier.cpp
+@@ -24,12 +24,18 @@
+ 
+ #include <QSocketNotifier>
+ 
++#ifdef __HAIKU__
++#  include <posix/signal.h>
++#  include <posix/sys/socket.h>
++#  include <unistd.h>
++#else
```

Comment:
`posix` in header path should not be needed. So, what's the problem here? Do we not have `sys/signal.h`? Just ifdef that one out then, don't duplicate this.